src/panel/context/Explorer/ast/variables.ts

```diff
@@ -0,0 +1,72 @@
+/** Copied from https://github.com/FormidableLabs/urql-exchange-graphcache/blob/master/src/ast/variables.ts */
+
+import { FieldNode, ValueNode, OperationDefinitionNode, Kind } from "graphql";
+
+import { getName } from "./node";
+import { Variables } from "./types";
+
+/** Evaluates a given ValueNode to a JSON value taking vars into account */
+export const evaluateValueNode = (node: ValueNode, vars: Variables): any => {
```

Comment:
In graphcache this was replaced replaced with: valueFromASTUntyped from the graphql package

https://github.com/FormidableLabs/urql-exchange-graphcache/pull/82/files#diff-850cd894e021a8dd8a5952f27d5e730fR68